addons/web/static/lib/odoo_ui_icons/style.css

```diff
@@ -7,6 +7,7 @@
 }
 
 .oi {
+  display: inline-block;
```

Comment:
The `oi` icons need `display: inline-block;` to handle RTL, just like the `fa` icons. 
Previously, only icons using `oi-fw` could handle RTL.

Moving the `display` property to `oi` is mandatory to correctly manage RTL on all `oi` icons.

What do you think @edi-odoo @stefanorigano ?

Reply:
It makes sense to me to have the same styling as font-awesome 🤷‍♀️

Reply:
I think that the only reason why we didn't do it before it's because of some weird `button.btn.oi` elements... now these should all be gone now with milk 👍